### ISSUE TITLE
.getUsagePeriods() throws a 404

### ISSUE THREAD

**Issue by michelbitter** [contentful/contentful-management.js#222]
Hey Contentful,

I was working on #221 when I found out that the function `getUsagePeriods` throws always an HTTP error 404. No matter what I do.

Code example:

```
import * as contentful from 'contentful-management'

const connection = contentful.createClient({
  accessToken: 'xxxxxxxx',
})

console.log(connection.getUsagePeriods('xxxxxxxx'))
```

Error Message:
```
(node:33602) UnhandledPromiseRejectionWarning: NotFound: {
  "status": 404,
  "statusText": "Not Found",
  "message": "The resource could not be found.",
  "details": {},
  "request": {
    "url": "https://api.contentful.com:443/organizations/xxxxxxxx/usage_periods",
    "headers": {
      "Accept": "application/json, text/plain, */*",
      "Content-Type": "application/vnd.contentful.management.v1+json",
      "X-Contentful-User-Agent": "sdk contentful-management.js/5.2.1; platform node.js/v10.16.1; os macOS/18.7.0;",
      "Authorization": "Bearer ...xxxxx",
      "user-agent": "node.js/v10.16.1",
      "Accept-Encoding": "gzip",
      "x-contentful-enable-alpha-feature": "usage-insights"
    },
    "method": "get"
  },
  "requestId": "962a530803eacd83f59311980b5763bb"
}
    at errorHandler (/Users/michelbitter/Development/contentful-management.js/dist/webpack:/contentfulManagement/error-handler.js:60:17)
    at process._tickCallback (internal/process/next_tick.js:68:7)
(node:33602) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 3)
(node:33602) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by Khaledgarbaya** [comment]
Hi @michelbitter this could be related to the type of account you have.

**Comment by michelbitter** [comment]
What do you mean precisely with "type of account"? I mean I'm the administrator of an organization so it is kind of weird that I can't see the billing periods while I can see them online in the GUI.

**Comment by Khaledgarbaya** [comment]
Fixed in #221
